packaging/windows/windows_msi.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi'>
+	<Product 
+		Name='lfs-s3' 
+		Manufacturer='Nicolas Graves'
+		Id='3f3987e3-b185-429a-86b3-00dd0ad27df1' 
+		UpgradeCode='98543bff-32f7-4ee9-ad4f-47697e2723b2'
```

Comment:
Sorry for the delay, been really busy.  These UUIDs are manually generated, but only need to be once.  The single exception is on line 13 with `Id='*'`.  `*` gets expanded into a UUID during the torch build phase.  If you add more components going forward, they'd need unique UUIDs, but for just having a single installable MSI, this is sufficient.

Note, this doesn't include creating MSP files which are patch files for upgrading from one version to the next, so consumers of these packages would need to uninstall their current version and install new ones.  I don't imagine this will be much of a barrier, but it is worth noting.